cloud_dashboard/src/organisms/CallbackForm.tsx

```diff
@@ -51,8 +52,19 @@ const CallbackForm = () => {
   }, []);
 
   return <form>
-      <h2>Please waiting...</h2>
-    </form>;
+    <div style={{ display: 'flex' }}>
+      <Loader
+        type="Grid"
+        color="#00BFFF"
+        height="3rem"
+        width="3rem"
+      />
+      <span style={{
+        fontSize: '2rem',
+        marginLeft: '1rem'
+      }}>One moment, please...</span>
```

Comment:
@RyoYamashita765 What about this?

```suggestion
        marginLeft: '2rem'
      }}>Orchestrating the clouds...</span>
```